### ISSUE TITLE
CustomBodyPoints

### ISSUE THREAD

**Issue by ShyamKrishna122** [farazBhatti/Human-Body-Measurements-using-Computer-Vision#26]
How was the custom body points.txt generated and what is the use of it? Will it work for all the images? If not, how to generate custom body points.txt for a given 2D image?

**Comment by jpcurbelo** [comment]
Hi, @ShyamKrishna122. Please, take a look at the discussion in issue #16 and let me know if you would need more details. Good luck!

**Comment by ShyamKrishna122** [comment]
So can we use the same custom body points,txt for finding anybody's body
measurements or will it differ according to the obj file generated? Thank
you for the help.

**Comment by jpcurbelo** [comment]
Yes, this is how I did. The meshes in the SPRING database are generated following the same pattern. I picked one female and one male model close to the "average" body and extracted the control points for those models. Then, I used the sets for all the female and male subjects, respectively. I compared the distributions to ANSUR II and they are close in shape, so I assumed that the method to extract the measurements is "good enough".
